doc/glossary.rst

```diff
@@ -4,6 +4,10 @@
 
 .. glossary::
 
+        Application
```

Comment:
IMHO the description fits _Client_ better than _Application_

Reply:
You're right, but the docs use "application". I think I'll alter this a bit to say "more properly called a 'client', an application..." and then I'll add another headword (that headword will be 'client') in which I'll try to disabuse the reader of this misunderstanding.